Tasks/Common/VstsAzureHelpers_/Utility.ps1

```diff
@@ -15,8 +15,9 @@
         $certificate.Import($pfxFilePath, $pfxFilePassword, [System.Security.Cryptography.X509Certificates.X509KeyStorageFlags]::PersistKeySet)
     }
     else {
+        $pfxFilePassword = [System.String]::Empty
         $bytes = [System.Convert]::FromBase64String($Endpoint.Auth.Parameters.Certificate)
-        $certificate.Import($bytes)
+        $certificate.Import($bytes, $pfxFilePassword, [System.Security.Cryptography.X509Certificates.X509KeyStorageFlags]::PersistKeySet)
```

Comment:
As the workaroud mentioned in comments here :https://github.com/Azure/azure-powershell/issues/13117#issuecomment-701707875, the person is using [System.Security.Cryptography.X509Certificates.X509KeyStorageFlags]::Exportable. Can you please check if PersistKeySet must be used or Exportable here

Reply:
Exportable is used when certificate is exported to some other certificate format

> Exportable | 4 | Imported keys are marked as exportable.
> -- | -- | --
> PersistKeySet | 16 | The key associated with a PFX file is persisted when importing a certificate.

Cause of this issue is mentioned here. [Azure/azure-powershell#13146](https://github.com/Azure/azure-powershell/issues/13146)

> "If the script is loading a certificate from the certificate store and it starts failing after some unpredictable amount of time, it's that whatever loaded the certificate into the certificate store didn't use PersistKeySet and it finally got garbage collected, erasing the "temporary" private key file".
> 
> This behavior is described here:
> 
> https://docs.microsoft.com/en-us/troubleshoot/dotnet/framework/install-pfx-file-using-x509certificate
> 
> Cause
> 
> When the certificate is installed by using the X509Certificate or X509Certificate2 class, X509Certificate or X509Certificate2 by default creates a temporary container to import the private key. The private key is deleted when there's no longer a reference to the private key.
>